app/templates/core-data/admin-unit/index.hbs

```diff
@@ -56,44 +57,59 @@
                 />
               </:content>
             </Item>
-            <Item>
-              <:label>KBO nummer</:label>
-              <:content>{{if
-                  @model.kbo
-                  (kboFormat @model.kbo.localId)
-                  "Niet ingegesteld"
-                }}</:content>
-            </Item>
-            <Item>
-              <:label>NIS code</:label>
-              <:content>{{if
-                  @model.nis
-                  @model.nis.localId
-                  "Niet ingegesteld"
-                }}</:content>
-            </Item>
-            <Item>
-              <:label>OVO-nummer</:label>
-              <:content>{{if
-                  @model.ovo
-                  @model.ovo.localId
-                  "Niet ingegesteld"
-                }}</:content>
-            </Item>
+            {{#if @model.resultedFrom}}
+              <Item>
+                <:content>
+                  <AuHelpText @skin="tertiary" class="au-u-margin-top-none">
+                    Gewijzigd op
+                    {{date-format @model.resultedFrom.firstObject.date}}
+                  </AuHelpText>
+                </:content>
+              </Item>
+            {{/if}}
+            {{#if @model.kbo}}
+              <Item>
+                <:label>KBO nummer</:label>
+                <:content>
+                  {{kbo-format @model.kbo.localId}}
+                </:content>
+              </Item>
+            {{/if}}
+            {{#if @model.ovo}}
+              <Item>
+                <:label>OVO nummer</:label>
+                <:content>
+                  {{@model.ovo.localId}}
+                </:content>
+              </Item>
+            {{/if}}
+            {{#if @model.nis}}
+              <Item>
+                <:label>NIS</:label>
+                <:content>
+                  {{@model.nis.localId}}
+                </:content>
+              </Item>
+            {{/if}}
+            {{#if @model.sharepoiint}}
+              <Item>
+                <:label>Sharepoint referentie
+                </:label>
+                <:content>
+                  {{!-- <AuLinkExternal
```

Comment:
Remove commented code if it's no longer needed

Reply:
This was intentional. We do not seem to have all the data present. OP shows a web link. I kept the old code in comment for this reason. But I'll remove it.